app/src/main/res/drawable/ic_new_pin.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="10dp"
+    android:height="10dp"
+    android:viewportWidth="10"
+    android:viewportHeight="10">
+    <path
+        android:fillColor="@color/foundation_contrast_normal_theme"
```

Comment:
Rather than adding a new variable that is only used here we can just keep using `top_site_pin_icon_background_tint` although perhaps with an updated name. We should also make to update or remove it.